prow/logrusutil/logrusutil.go

```diff
@@ -61,5 +61,6 @@ func (d *DefaultFieldsFormatter) Format(entry *logrus.Entry) ([]byte, error) {
 		Time:    entry.Time,
 		Level:   entry.Level,
 		Message: entry.Message,
+		Caller:  entry.Caller,
```

Comment:
How does this end up being conditional?

Reply:
@stevekuznetsov As from Logrus code these two checks does: https://github.com/sirupsen/logrus/blob/e1e72e9de974bd926e5c56f83753fba2df402ce5/entry.go#L186 

So, the `ReportCaller` must be true (false by default) and the `entry.Caller` shouldn't be nil if these conditions are met then Logrus will print the `func` and the `line`.

That's why I added the `entry.Caller` but it's still conditional.